opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -65,21 +53,21 @@ impl EventVisitor {
 
         if let Some(module_path) = meta.module_path() {
             self.log_record_attributes
-                .push(("log.module.path".into(), module_path.to_owned().into()));
+                .push(("code.namespace".into(), module_path.to_owned().into()));
         }
 
         if let Some(filepath) = meta.file() {
             self.log_record_attributes
-                .push(("log.source.file.path".into(), filepath.to_owned().into()));
+                .push(("code.filepath".into(), filepath.to_owned().into()));
             self.log_record_attributes.push((
-                "log.source.file.name".into(),
+                "code.filename".into(),
```

Comment:
Do we need `code.filename` as it is not part of semantic convention, and `code.filepath` contains the fully qualified path with filename ?

Reply:
It is easier to do an exact match with only the name of the file. Remember also the path can change between compilations. Also, search engines like ElasticSearch has a column type called keyword for exact match that is more efficient than text search.